atlite/cutout.py

```diff
@@ -354,6 +369,18 @@ def grid_cells(self):
 
     @CachedAttribute
     def grid(self):
+        """
+        Cutout grid with coordinates and geometries.
+
+        The coordinates are placed in the middle of the grid cells.
```

Comment:
```suggestion
        The coordinates represent the centers of the grid cells.
```

For consistency with doc.